Workbooks/LogAnalytics Workspace/Overview/LA Workspace Insights Overview.workbook

```diff
@@ -0,0 +1,590 @@
+{
```

Comment:
Will this just be a separate workbook, or would it be incorporated into AtResource?

Reply:
Just like the others - Can stand on its own, but will also be incorporated into AtResource